spec/book_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './book'
+
+# Test on Book Class
+describe Book do
+  describe '#initialize' do
+    it 'sets the title and author attributes' do
+      book = Book.new(title: 'The Great Gatsby', author: 'F. Scott Fitzgerald')
+      expect(book.title).to eq('The Great Gatsby')
+      expect(book.author).to eq('F. Scott Fitzgerald')
+    end
+
+    it 'initializes the rentals attribute as an empty array' do
+      book = Book.new(title: 'The Great Gatsby', author: 'F. Scott Fitzgerald')
+      expect(book.rentals).to eq([])
+    end
+  end
+end
```

Comment:
- [ ] All public methods should be tested. Kindly add tests for the `add_rental` in your Book class.

Reply:
Thank you. I mistakenly missed out on that one.